tests/stubtest_allowlists/py3_common.txt

```diff
@@ -374,6 +374,10 @@ threading.Condition.release
 multiprocessing.dummy.Condition.acquire
 multiprocessing.dummy.Condition.release
 
+# Some of these take a "_warn" parameter at runtime
+# for technical reasons; no need to expose it in the stub.
+asyncio\..*\.__del__
```

Comment:
I'd actually prefer it if we tediously enumerated all the methods that need to be allowlisted here, since using wildcards in allowlist entries like this often comes back to bite us later